src/meta/processors/sessionMan/SessionManagerProcessor.cpp

```diff
@@ -0,0 +1,119 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "meta/processors/sessionMan/SessionManagerProcessor.h"
+
+namespace nebula {
+namespace meta {
+
+void CreateSessionProcessor::process(const cpp2::CreateSessionReq& req) {
+    folly::SharedMutex::WriteHolder wHolder(LockUtils::sessionLock());
+    const auto& user = req.get_user();
+    auto ret = userExist(user);
+    if (ret != cpp2::ErrorCode::SUCCEEDED) {
+        handleErrorCode(ret);
+        onFinished();
+        return;
+    }
+
+    cpp2::Session session;
+    // The sessionId is generated by microsecond timestamp
+    session.set_session_id(time::WallClock::fastNowInMicroSec());
+    session.set_create_time(session.get_session_id());
+    session.set_update_time(session.get_create_time());
+    session.set_user_name(user);
+    session.set_graph_addr(req.get_graph_addr());
+    session.set_client_ip(req.get_client_ip());
+
+    std::vector<kvstore::KV> data;
+    data.emplace_back(MetaServiceUtils::sessionKey(session.get_session_id()),
+                      MetaServiceUtils::sessionVal(session));
+    resp_.set_session(session);
+    doSyncPutAndUpdate(std::move(data));
```

Comment:
Does session involves updating meta info? I think `doSyncPut` is enough. `doSyncPutAndUpdate` will cause every meta client load meta info.

Reply:
Thanks for your comment. Previously, graphd process session information from the cache, but it was not much use. so here is no need `doSyncPutAndUpdate` interface.